.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+*.log
```

check .gitignore with `git status` command
```suggestion
logs/*
```